zipkin-lens/.env.development

```diff
@@ -1,2 +0,0 @@
-BASE_PATH=/zipkin
```

Comment:
none of these files are needed for  `npm run build`, `npm run test` or `npm run start`, so removed all of them to avoid confusion of explaining why they are here.